W3ChampionsStatisticService/Admin/GlobalChatBan.cs

```diff
@@ -53,4 +56,5 @@ public class ChatBanPutDto
 {
     public string battleTag { get; set; }
     public string expiresAt { get; set; }
+    public string author { get; set; }
```

Comment:
Can we infer the author from the auth data, rather than it being an input field?